app.py

```diff
@@ -849,17 +896,55 @@ def update_street_name(*args):
         *selected_streets, orientation = args
         street = selected_streets[list(SELECTED_STREET_DIVS.keys()).index(orientation)]
         try:
+            
             from_to = BASELINE[(BASELINE['street'] == street[0]) &
                                (BASELINE['direction'] == DIRECTIONS[orientation][dir_id])][['from_intersection',
                                                                                'to_intersection']].iloc[0]
+            for n, i in enumerate(from_to):
```

Comment:
This data formatting exercise should be done in PostgreSQL

Reply:
Fixed with https://github.com/CityofToronto/bdit_traffic_dashboard/pull/36/commits/c0f9c109563d980db4e80b08c6b08903a5662dc6